.github/workflows/IO_Iceberg.yml

```diff
@@ -71,8 +71,7 @@ jobs:
       github.event_name == 'push' ||
       github.event_name == 'pull_request_target' ||
       (github.event_name == 'schedule' && github.repository == 'apache/beam') ||
-      github.event_name == 'workflow_dispatch' ||
-      github.event.comment.body == 'Run Java_Amqp_IO_Direct PreCommit'
```

Comment:
We can keep this. The phrase trigger for PreCommit is currently working

Reply:
Done